lightning/src/ln/offers_tests.rs

```diff
@@ -205,12 +205,12 @@ fn extract_invoice_request<'a, 'b, 'c>(
 	}
 }
 
-fn extract_invoice<'a, 'b, 'c>(node: &Node<'a, 'b, 'c>, message: &OnionMessage) -> Bolt12Invoice {
+fn extract_invoice<'a, 'b, 'c>(node: &Node<'a, 'b, 'c>, message: &OnionMessage) -> (Bolt12Invoice, Option<BlindedPath>) {
 	match node.onion_messenger.peel_onion_message(message) {
-		Ok(PeeledOnion::Receive(message, _, _)) => match message {
+		Ok(PeeledOnion::Receive(message, _, reply_path)) => match message {
```

Comment:
As a follow-up PR, could you start including a reply path with invoices for offers? We should have the capability now, but we just haven't updated `ChannelManager`'s implementation of `OffersMessageHandler` yet.

Reply:
Sure! Added it to the task list! 🚀

Reply:
#3163